deploy/vagrant/provision/roles/mesos/files/start_slave.sh

```diff
@@ -5,4 +5,4 @@ set -e
 cd /mesos/build
 MESOS_LOGS_DIR=/mesos/logs
 mkdir -p "${MESOS_LOGS_DIR}"
-./bin/mesos-slave.sh --work_dir=/mesos/workdir/slave --port=1235 --master=TachyonMaster:1234 > "${MESOS_LOGS_DIR}/slave.out" 2>&1 &
+./bin/mesos-slave.sh --work_dir=/mesos/workdir/slave --master=TachyonMaster:50050 > "${MESOS_LOGS_DIR}/slave.out" 2>&1 &
```

Comment:
If the port option is not set, a port number is picked dynamically, which is what we want to for the worker.